pallets/automation-time/src/types.rs

```diff
@@ -213,6 +214,7 @@ pub struct Task<AccountId, Balance> {
 	pub provided_id: Vec<u8>,
 	pub schedule: Schedule,
 	pub action: Action<AccountId, Balance>,
+	pub cancel_upon_errors: Vec<String>,
 }
```

Comment:
The name of `cancel_upon_errors` was not well thought. Let’s call it `abortErrors` (abort_errors) to be more intuitive.

Any task with a Recurring schedule should include this parameter.

The detailed explanation is in #393